### PR TITLE
web/portal: Added Service entity to platform portal

### PR DESCRIPTION
**Pull request by fmendozzac**
#### Type Of Change 
- [ ] Small bug fix
- [x] New feature or enhancement
- [ ] Breaking change

#### Checklist:
- [x] Commits are named and have tag following [commit rules](https://github.com/irontec/ivozprovider/blob/bleeding/doc/dev/en/commits.md)
- [x] Commits are split per component (schema, web/admin, kamusers, agis, ..)
- [x] Changes have been tested locally
- [ ] Fixes an existing issue (Fixes #XXXX) 
- [ ] Upport from existing Pull request #XXXX

#### Description
Added Service entity to platform portal

### CODE REVIEW

**Comment by ironArt3mis** [comment]
Rebase completed.

File: web/portal/platform/src/entities/Service/SelectOptions.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  callback,
+  cancelToken,
+}): Promise<unknown> => {
+  const entities = store.getState().entities.entities;
+  const Service = entities.Service;
+
+  return defaultEntityBehavior.fetchFks(
+    Service.path,
+    ['id'],
+    (data) => {
```
Comment (mmadariaga) [inline web/portal/platform/src/entities/Service/SelectOptions.ts:16]:
```suggestion
    (data: EntityValues) => {
```

File: web/portal/platform/src/entities/Service/Service.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  },
+};
+
+const Service: EntityInterface = {
+  ...defaultEntityBehavior,
+  icon: HomeRepairServiceIcon,
+  iden: 'Service',
+  title: _('Service', { count: 2 }),
+  path: '/services',
+  toStr: (row: EntityValues) => row.name as string,
```
Comment (mmadariaga) [inline web/portal/platform/src/entities/Service/Service.tsx:45]:
```suggestion
  toStr: (row: ServicePropertyList<EntityValue>) => row.name as string,
```

**Review by mmadariaga — CHANGES_REQUESTED** [review]

File: web/portal/platform/src/entities/Service/SelectOptions/SelectOptions.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  cancelToken,
+}): Promise<unknown> => {
+  const entities = store.getState().entities.entities;
+  const Service = entities.Service;
+  const language = getI18n().language.substring(0, 2);
+
+  return defaultEntityBehavior.fetchFks(
+    Service.path + `?_order[name.${[language]}]=ASC`,
+    ['id', 'name'],
+    (data) => {
```
Comment (mmadariaga) [inline web/portal/platform/src/entities/Service/SelectOptions/SelectOptions.ts:18]: Missing hint

File: web/portal/platform/src/entities/Service/SelectOptions/UnassignedServiceSelectOptions.ts [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+  let path = `${Service.path}/unassigned`;
+  if (includeId) {
+    path += `?_includeId=${includeId}`;
+  }
+
+  return defaultEntityBehavior.fetchFks(
+    path,
+    ['id', 'name'],
+    (data) => {
```
Comment (mmadariaga) [inline web/portal/platform/src/entities/Service/SelectOptions/UnassignedServiceSelectOptions.ts:30]: Missing hint

**Review by mmadariaga — CHANGES_REQUESTED** [review]

**Review by mmadariaga — APPROVED** [review]

**Pull request merged**